apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -370,6 +372,22 @@ export default class ScriptEditor extends React.Component {
                 </p>
               </HelpTip>
             </label>
+            <label>
+              Can be translated (only check this if the script is stable)
+              <input
+                name="is_translateable"
+                type="checkbox"
+                defaultChecked={this.props.isTranslateable}
+                style={styles.checkbox}
+              />
+              <HelpTip>
+                <p>
+                  If checked, this unit will be included in our translation
+                  pipeline. Major changes to this unit after translation has
+                  started can break translations.
```

Comment:
I'm a bit worried about this. We have recently had some issues with scripts being confirmed as being ready for translation, but then major changes coming in afterwards. Are we confident that the curriculum team has been adequately prepared for this responsibility?

Reply:
see other comment for one protection we're adding right now. there are lots of other things a curriculum team member could still change, though. one idea for increased certainty would be to only show this option if the script is already marked as "is_stable", which only happens after new curriculum has been "fully launched". however that only really makes sense for versioned courses, so not sure it is a good blanket solution. happy to discuss this more.

Reply:
Yeah originally I had this option greyed out until the script was marked as stable, which seems like a reasonable first step guard for versioned courses (though I didn't implement a corresponding validation on the model itself). I'm not really sure what a good solution is for non-versioned courses...